module/users/helper.js

```diff
@@ -493,4 +490,106 @@ module.exports = class UsersHelper {
         })
     }
 
+      /**
+      * User targeted solutions.
+      * @method
+      * @name solutions
+      * @param {String} programId - program id.
+      * @param {Object} requestedData requested data.
+      * @param {Object} token Token.
+      * @param {String} pageSize page size.
+      * @param {String} pageNo page no.
+      * @param {String} search search text.
+      * @returns {Object} targeted user solutions.
+     */
+
+    static solutions( programId,requestedData,token,pageSize,pageNo,search ) {
+        return new Promise(async (resolve, reject) => {
+            try {
+                
+                let autoTargetedSolutions = 
```

Comment:
@aman-tunerlabs - consider moving the logic to Kendra as it deals with all kinds of solutions.